aten/src/ATen/CMakeLists.txt

```diff
@@ -37,27 +48,52 @@ set_bool(AT_CUSPARSELT_ENABLED CAFFE2_USE_CUSPARSELT)
 set_bool(AT_HIPSPARSELT_ENABLED CAFFE2_USE_HIPSPARSELT)
 
 configure_file(Config.h.in "${CMAKE_CURRENT_SOURCE_DIR}/Config.h")
-# TODO: Do not generate CUDAConfig.h for ROCm BUILDS
```

Comment:
Don't like this change.
cmake-format knows about TODOs, but thinks they have to come with an assignee. This regex would need a patch: https://github.com/cheshirekow/cmake_format/blob/eff5df1f41c665ea7cac799396042e4f406ef09a/cmakelang/markup.py#L12